src/components/WarningModal.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import { Modal } from 'antd';
+
+interface Props {
+  isMobile: boolean;
+}
+
+export const WarningModal: React.FC<Props> = (props: Props) => {
+  const { isMobile } = props;
+  const [isVisible, setVisible] = useState(true);
```

Comment:
```suggestion
  const [isVisible, setVisible] = useState(false);
```
better to start off as false so that when the website renders the page with this component, the modal component will not be visible. 

**For example:**
If the website is opened as a desktop version, this component will pop up because the initial value of this state is true.

Reply:
:+1: